requirements.txt

```
@@ -8,9 +8,9 @@ opt_einsum
 pathos>=0.2.7
 pylatexenc>=2.10
 pyscf>=2.0.1
-qiskit>=0.39.0,<1.0.0
+qiskit>=1.0.0
 recommonmark
-qiskit_ibm_runtime==0.20.0
+qiskit-ibm-runtime==0.20.0
 qiskit-aer==0.13.3
```

Thank you for reviewing! I will commit the change once the test is fixed.
```suggestion
qiskit-aer>=0.13.3
```